forui/lib/l10n/f_af.arb

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Are these all automatically generated?

Reply:
Yup!

Reply:
The only one that isn't is the template, `f_en.arb`.